C-T-M.cpp

```diff
@@ -293,21 +556,84 @@ int main()
                     }
                 }
                 break;
+            }
 
             case 2:
+            {
                 cout << "\nHUMAN SUBJECTS DASHBOARD\n";
 
-                break;
-
-            default:
-                cout << "\nInvalid choice !\n";
-                exit(0);
-            }
-        }
-    }
-
+                for(;;){
+                    cout<<"Enter the drug name\n";

```

Comment:
its taking the drug name, even if its records are not there in drug.txt,thats fine rgt?